src/emulator/auth/operations.ts

```diff
@@ -2543,6 +2556,123 @@ function parsePendingCredential(
   return { user, signInProvider };
 }
 
+function createTenant(
+  state: ProjectState,
+  reqBody: Schemas["GoogleCloudIdentitytoolkitAdminV2Tenant"]
+): Schemas["GoogleCloudIdentitytoolkitAdminV2Tenant"] {
+  assert(state instanceof AgentProjectState, "((Can only create tenant in agent project.))");
+
+  const formattedPhoneNumbers: { [key: string]: string } = {};
```

Comment:
This can be a `Record<string, string>` type: https://www.typescriptlang.org/docs/handbook/utility-types.html#recordkeystype

Reply:
Sick, good call, thanks!